docs/release_notes/index.rst

```diff
@@ -7,6 +7,7 @@ called "Plankton").
 .. toctree::
     :maxdepth: 1
 
+    release_1_1_0
```

Comment:
In an attempt to avoid confusion we did not include this in the index before the actual release. See comment below.